frontend/src/App.jsx

```diff
@@ -11,7 +11,8 @@ function App() {
   const [count, setCount] = useState(0)
   const [message, setMessage] = useState("Test Message")
   const { logout, session } = useAuth()
-  const [authorized, setAuthorized] = useState("Test Authorization"); 
+  const [authorized, setAuthorized] = useState("Test Authorization");
```

Comment:
Could we perhaps initialize these with better values? Currently the UI is a bit confusing, as there are two elements (a button and a text) both with 'Test Authorization' and 'Test Database'. Would perhaps change it to empty value instead for initial state, and then update it with the repsonse or error returned from the auth or database check.